lib/geo/json.ex

```diff
@@ -30,7 +30,7 @@ defmodule Geo.JSON do
   @doc """
   Takes a map representing GeoJSON and returns a Geometry.
   """
-  @spec decode!(map()) :: Geo.geometry() | no_return
```

Comment:
no_return should be used when the function _never_ returns (always raises, throws, endlessly loops, etc). Otherwise we'd need to annotate almost every function as they may e.g. raise an exception for invalid args, function clause, etc!

Reply:
Ah got it. That makes sense